recommender/api/schemas/database_dump/user.py

```diff
@@ -13,6 +13,12 @@
             description="The unique identifier of a the user",
             example=1234,
         ),
+        "aai_token": fields.String(
+            required=True,
+            title="AAI Token",
+            description="AAI token of the user",
+            example="abcdefghijklmnop",
```

Comment:
An example may be better. Look [recommender/api/schemas/recommendation.py](https://github.com/cyfronet-fid/recommender-system/pull/394/files#diff-a395fd459f377dc7e7a1575bf337d5e2bb9a7eafbbd5e77070ce7d9b802fbbe1)

            example="zooehewplbfxqtnxpspzsgrmeadoynbygyvisuvzst"
            + "dgenzxklzuwfdjndtzitiz@egi.eu",